src/Impostor.Server/Net/State/Game.Api.cs

```diff
@@ -20,24 +23,45 @@ public void BanIp(IPAddress ipAddress)
             _bannedIps.Add(ipAddress);
         }
 
-        // TODO This no longer does anything, it was replaced by LogicOptions
         public async ValueTask SyncSettingsAsync()
         {
             if (Host?.Character == null)
             {
-                throw new ImpostorException("Attempted to set infected when the host was not spawned.");
+                throw new ImpostorException("Attempted to change settings when the host was not spawned.");
             }
 
-            using (var writer = StartRpc(Host.Character.NetId, RpcCalls.SyncSettings))
+            var gameManager = FindObjectByType<InnerGameManager>();
```

Comment:
Just cache it like every other netobject in `Game.OnSpawnAsync`?

Reply:
Right, I forgot about that.